doc/tasks/stylelint.md

```diff
@@ -0,0 +1,187 @@
+# Stylelint ![fixer](https://img.shields.io/badge/-fixer-informational)
+
+[Stylelint](https://stylelint.io/) is a modern linter that helps you avoid errors and enforce conventions in your styles.
+
+## NPM
+If you'd like to install it globally:
+```bash
+npm -g stylelint
+```
+
+If you'd like install it as a dev dependency of your project:
+```bash
+npm install --save-dev stylelint
+```
+
+Done. See the Stylelint [Getting Started](https://stylelint.io/user-guide/get-started) guide for more info.
+
+## Config
+It lives under the `stylelint` namespace and has the following configurable parameters:
+
+```yaml
+# grumphp.yml
+grumphp:
+    tasks:
+        stylelint:
+            bin: node_modules/.bin/stylelint
+            triggered_by: [css, scss, sass, less, sss]
+            whitelist_patterns:
+                - /^resources\/css\/(.*)/
+            config: ~
+            config_basedir: ~
+            ignore_path: ~
+            ignore_pattern: ~
+            syntax: ~
+            custom_syntax: ~
+            ignore_disables: ~
+            disable_default_ignores: ~
+            cache: ~
+            cache_location: ~
+            formatter: ~
+            custom_formatter: ~
+            quiet: ~
+            color: ~
+            report_needless_disables: ~
+            report_invalid_scope_disables: ~
+            report_descriptionless_disables: ~
+            max_warnings: ~
+            output_file: ~
+```
+
+**bin**
+
+*Default: null*
+
+The path to your stylelint bin executable. Not necessary if stylelint is in your $PATH. Can be used to specify path to project's stylelint over globally installed stylelint.
+
```

Comment:
Maybe we could add here that it is possible to add node_modules/.bin to the grumphp paths in order for it to be automatically discovered as well:

```yaml
grumphp:
    environment:
        paths:
            - node_modules/.bin
```